src/System.Windows.Forms.Primitives/src/Interop/Shell32/Interop.SHCreateShellItem.cs

```diff
@@ -16,7 +16,7 @@ internal static partial class Shell32
         public static IShellItem GetShellItemForPath(string path)
         {
             uint zero = 0;
-            if (SHILCreateFromPath(path, out IntPtr pidl, ref zero).Succeeded())
+            if (SHParseDisplayName(path, IntPtr.Zero, out IntPtr pidl, 0, out zero).Succeeded())
```

Comment:
If you're doing it as an `out` param, you can do this:
```suggestion
            if (SHParseDisplayName(path, IntPtr.Zero, out IntPtr pidl, 0, out uint _).Succeeded())
```